src/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -752,6 +756,14 @@ internal static byte[] ConvertPublicKeyToPublicKeyToken(byte[] publicKey)
         [RuntimeImport(RuntimeLibrary, "RhpArrayClear")]
         internal static extern bool TryArrayClear(Array array, int index, int length);
 
+        [RuntimeImport(".", "RhpGetModuleSection")]
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        internal static extern IntPtr GetModuleSection(IntPtr module, int readyToRunSectionId, out int length);
+
+        [RuntimeImport(".", "RhpCreateModuleManager")]
```

Comment:
This looks redundant with RhpCreateTypeManager/RhGetModuleSection. Did you forget to remove it?